sdk/communication/azure-communication-messages/CHANGELOG.md

```diff
@@ -22,5 +12,14 @@ This is the GA release of Azure Communication Messages Python SDK. For more info
 - Using `MessageTemplateClient`
   - Get list of all WhatsApp templates.
 
+## 1.0.1 (2024-10-10)
```

Comment:
Generally, keep latest version changes log entry at the top. I see same for SDK module.

Reply:
For Python , This need to be at the bottom. Build is failing if I am moving it anywhere else.

Reply:
Correct, this should at the top, and the other langauges are using 1.1.0 as the version numbe rinstead of 1.0.1, is that something Python specifically adheres to? If not, could we align with the same version number?

Reply:
I see other SDK module having latest version entry at the top. Below is some example

https://github.com/Shamkh/azure-sdk-for-python/blob/7e904912629037ca8733cda82dc1e7cde4a091c6/sdk/communication/azure-communication-sms/CHANGELOG.md?plain=1#L3

https://github.com/Shamkh/azure-sdk-for-python/blob/7e904912629037ca8733cda82dc1e7cde4a091c6/sdk/communication/azure-communication-phonenumbers/CHANGELOG.md?plain=1#L3

I am not sure why our module alone will have build failure when other module has same structure.

Reply:
version is updated to 1.1.0